software/seafile.yml

```
@@ -10,6 +10,9 @@ platforms:
   - C
 tags:
   - File Transfer & Synchronization
+  - knowledge-management-tools
+  - office-suites
+  - wikis
```

```suggestion
  - Knowledge Management Tools
  - Office Suites
  - Wikis
```
This is the appropriate case, not the file names of the tags.

---

Also please remove the commit that involves adding the tags, it'll be a duplicate.